images/expansionhunter_bw2/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG VERSION=${VERSION:-5.0.0-bw2}
+
+# Install Git to clone the repository
+RUN apt-get update && \
+    apt-get install -y \
+    cmake \
+    g++ \
+    git \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    liblzma-dev \
+    libssl-dev \
+    zlib1g-dev
+
+# Clone the repository, jump into it
+# build the ExpansionHunter executable, +x, move to /usr/bin
+RUN git clone https://github.com/bw2/ExpansionHunter && \
+    cd ExpansionHunter && \
```

Comment:
At the moment, this just uses **master** from bw2's fork. This may move in future, so for reproducibility we need to identify the particular commit we currently want. So please checkout a commit explicitly here (between the `cd` and `mkdir build`):

```
    git checkout $VERSION && \
```

The commit to use is the current head of bw2's **master**, which is b85cba004fba4e0223e632fae576bc7fad0da804.

And then use that instead of `5.0.0-bw2` in the `ARG VERSION` line above and in _images.toml_.

This image needs a particular untagged commit hash, so the version number bits should look like the images for _haplocheckcli_, _saige-qtl_, and _stripy_.